services/notification/notifiers/tests/unit/test_status.py

```diff
@@ -2584,6 +2587,11 @@ def test_build_payload_with_multiple_changes(
         expected_result = {
             "message": "3 files have indirect coverage changes not visible in diff",
             "state": "failure",
+            "included_helper_text": {
+                INDIRECT_CHANGES_KEY: HELPER_TEXT_MAP[INDIRECT_CHANGES_KEY].format(
+                    context="changes", notification_type="status"
```

Comment:
just like with the other text, I would prefer the actual text to be asserted here.
otherwise the test is just a mirror of the code doing the formatting.